src/helpers.tsx

```diff
@@ -0,0 +1,24 @@
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
+
+export const filteredTodos = (todos: Todo[], status: Status) => {
+  const preparedTodos = [...todos];
+
+  switch (status) {
+    case Status.all: {
+      return preparedTodos;
+    }
```

Comment:
This case returns the same as the `default` block, so you should either combine the cases or remove this case